README.md

```diff
@@ -1,3 +1,32 @@
+
+[comment]: # (Licensed to the Apache Software Foundation [ASF] under one)
+
+[comment]: # (or more contributor license agreements.  See the NOTICE file)
```

Comment:
For md file, you can follow this comment style, which is easier.

https://raw.githubusercontent.com/apache/incubator-singa/master/README.md

Reply:
pls update it.
thanks